index.html

```diff
@@ -266,5 +266,45 @@ <h3>Skills</h3>
         <div class="line"></div>
       </div>
     </section>
+
+    <section class="contact">
+      <h2>Contact me</h2>
```

Comment:
- [optional] Great job making the changes. Please kindly add some space between the contact section and the skill part like in the original design. Remember this is only optional because it was not mentioned but it is very important for your design.

your design             |  figma design
:-------------------------:|:-------------------------:
![Screenshot (910)_LI](https://user-images.githubusercontent.com/65298048/228500130-1cad2dee-3ec1-4934-a8fe-525245b13ef2.jpg)  |  ![Screenshot (911)](https://user-images.githubusercontent.com/65298048/228500230-09f790eb-da3e-442c-be30-b192955472a0.png)